packages/xml-views-validation/src/validators/attributes/invalid-boolean-value.ts

```diff
@@ -0,0 +1,52 @@
+import { XMLAttribute } from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { getPropertyByAttributeKey } from "@ui5-language-assistant/logic-utils";
+import { find, map } from "lodash";
+import { InvalidBooleanValueIssue } from "../../../api";
+import { isPossibleBindingAttributeValue } from "../../utils/is-binding-attribute-value";
+
+export function validateBooleanValue(
+  attribute: XMLAttribute,
+  model: UI5SemanticModel
+): InvalidBooleanValueIssue[] {
+  const actualAttributeValue = attribute.value;
+  const actualAttributeValueToken = attribute.syntax.value;
+  if (
+    actualAttributeValue === null ||
+    actualAttributeValueToken === undefined ||
+    isPossibleBindingAttributeValue(actualAttributeValue)
+  ) {
+    return [];
+  }
+
+  const ui5Property = getPropertyByAttributeKey(attribute, model);
+  const propType = ui5Property?.type;
+  if (propType?.kind !== "PrimitiveType" || propType.name !== "Boolean") {
```

Comment:
are there typos in the SDK? e.g "Boolean" vs "boolean"?
should we ignore case? or should the SDK be fixed?

Reply:
There are but it's handled in the semantic model, the strings used here (`"PrimitiveType"` and `"Boolean"`) are well-defined

Reply:
Although I'm thinking about making the primitive type names lowercase because that's generally how they are written in the api.json and the sdk